rust/agents/scraper/migration/src/m20240823_000006_add_injective_neutron_osmosis.rs

```diff
@@ -0,0 +1,138 @@
+use sea_orm::prelude::TimeDateTime;
+use sea_orm::ActiveValue::Set;
+use sea_orm::{ActiveModelTrait, DbErr, DeriveMigrationName, NotSet};
+use sea_orm_migration::{MigrationTrait, SchemaManager};
+use time::OffsetDateTime;
+
+use crate::async_trait;
+use crate::m20240823_000006_add_injective_neutron_osmosis::domain::ActiveModel;
+
+/// List of domain data we want to add into database.
+const DOMAINS: &[RawDomain] = &[
```

Comment:
fwiw to date our habits around adding domains haven't required us to create a new migration for new chains - calling it out not because I think our existing habits are good though 🙈

Our process for adding new chains has just been:
- add the new chain to the existing migration https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/rust/agents/scraper/migration/src/m20230309_000001_create_table_domain.rs#L16. We haven't actually applied this migration against the prod db since we initially stood it up. So this change is really only applied in e2e, or if we were to stand up a new scraper DB in prod.
- when adding new domains into our prod db, we run a SQL update query directly against the db to add them 😅

I'm very happy to change this process as obviously it's sketchy, but wondering if you'd prefer to adjust this process as part of your work here or to change this later. I'm fine with just adding these domains into the existing migration file (less duplicated code for the up and down logic) and then we can work with Paul to figure out a better setup for the future when it comes to prod db changes - but if you think it's cleaner in general to have separate migrations, I'm happy with that too

Reply:
I prefer to adjust it later. 😅